README.md

```diff
@@ -0,0 +1,71 @@
+# Apache Beam (incubating) website
+
+This is the website for [Apache Beam](http://beam.incubator.apache.org)
+(incubating).
+
+### About this site
+The Beam website is built using [Jekyll](http://jekyllrb.com/). Additionally,
+for additional formatting capabilities, this website uses
+[Twitter Bootstrap](http://getbootstrap.com/).
+
+This website is presently hosted at:
```

Comment:
Well, it is presently there. If the project graduates, it won't be anymore. :) The idea of calling it out as "presently" is to avoid any assumptions it will **always** be at this URL forever.